### PR TITLE
Changes after discussing standardization with the team

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

@dav3r and I discussed making some changes to the roles, policies, variables, etc. that will appear in every account subdirectory.  These are the changes appropriate to the "terraform" subdirectory.

## 💭 Motivation and Context

Ordnung muss sein!

## 🧪 Testing

I tested by applying the changes to the Terraform account.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: terraform/outputs.tf [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  value       = aws_iam_role.access_terraform_backend_role.arn
   description = "The ARN of the IAM role that allows sufficient access to the Terraform S3 bucket and DynamoDB table to use those resources as a Terraform backend."
 }
 
-output "create_role_arn" {
-  value       = aws_iam_role.create_role.arn
-  description = "The ARN of the IAM role that allows sufficient sufficient permissions to create all AWS resources in this account."
+output "provision_role_arn" {
+  value       = aws_iam_role.provision_account_role.arn
+  description = "The ARN of the IAM role that allows sufficient sufficient permissions to provision all AWS resources in this account."
```
Comment (dav3r) [inline terraform/outputs.tf]: Heads up: the word `sufficient` is duplicated in this description.

File: terraform/README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 "terraform" account.  It creates:
 
 * The S3 bucket used to store Terraform state.
 * The DynamoDB table used for Terraform state locking.
 * An IAM role that allows sufficient access to the Terraform S3 bucket
   and DynamoDB table to use those resources as a Terraform backend.
   This role also has a trust relationship with the users account.
-* An IAM role that allows sufficient sufficient permissions to create
-  all AWS resources in this account.  This role has a trust
+* An IAM role that allows sufficient sufficient permissions to
```
Comment (dav3r) [inline terraform/README.md]: Duplicated `sufficient` again  ♊️

Comment (jsf9k) [reply terraform/README.md]: Fixed in 2ec33a9.

File: terraform/outputs.tf (hunk 1 above)
Comment (jsf9k) [reply terraform/outputs.tf]: Fixed in 2ec33a9.

**Pull request merged**